src/pages/mentorship/mentorship.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLoaderData } from 'react-router-dom';
+import { PAGE_NAMES } from '@/shared/constants.ts';
+import { useTitle } from '@/shared/hooks/use-title';
+import { AboutMentors } from '@/widgets/about-mentors';
+import { Benefits } from '@/widgets/benefits';
+import { Breadcrumbs } from '@/widgets/breadcrumbs';
+import { HeroPage } from '@/widgets/hero-page';
+import { MemberActivity } from '@/widgets/member-activity';
+import { MentorsDocs } from '@/widgets/mentors-docs';
+import { MentorsRegister } from '@/widgets/mentors-register';
+import { MentorshipCourse, benefitMentorshipMentors } from 'data';
+
+const studyPathName = 'mentorship';
```

Comment:
On course pages, the path will also contain the course name `/mentorship/reactjs` and will require the first segment to be parsed.
This is a temporary solution so as not to delay the mentorship page - I've got an ishy refactoring this widget in the works right now.
Because of the overuse of the widget in the mentorship page, its types and parameters need to be changed.